cypress/e2e/app_tests/test1.cy.js

```diff
@@ -0,0 +1,13 @@
+describe('Form Layouts Test', () => {
+    it('Checks the Form Layouts page', () => {
+      cy.visit('/pages/layout/stepper');
+      cy.get('.step-content').first().should('include.text', 'Step content #1');
+      cy.contains('next').first().click();
+      cy.get('.step-content').first().should('include.text', 'Step content #2');
+      cy.contains('next').first().click();
+      cy.get('.step-content').first().should('include.text', 'Step content #3');
+      cy.contains('next').first().click();
+      cy.get('.step-content').first().should('include.text', 'Step content #4');
+      cy.contains('next').first().should('be.disabled');
```

Comment:
small improvement how to make it better 
```suggestion
      const arrEl = ['Step content #1', 'Step content #2', 'Step content #3', 'Step content #4'];

      arrEl.forEach(el => {
          cy.get('.step-content').first().should('include.text', el);
          el === 'Step content #4' ? cy.contains('next').first().should('be.disabled')
          :cy.contains('next').first().click();
      })

```